### ISSUE TITLE
Mentions plugin: the "focused" option should have an `aria-selected="true"` attribute

### ISSUE THREAD

**Issue by afercia** [draft-js-plugins/draft-js-plugins#1073]
The combobox ARIA widget is a bit complicated. There are now two different valid patterns, the "traditional" one from ARIA 1.0 and the new one from ARIA 1.1. The differences between the two patterns are very well described in the [ARIA Authoring Practices 1.1](https://www.w3.org/TR/wai-aria-practices-1.1/#combobox).

Regardless, in both patterns the "highlighted" option should get an `aria-selected="true"` attribute which is missing in the mentions plugin implementation.

References:

ARIA 1.0
https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-selected
> Used in Roles: ... option 

ARIA 1.1
https://www.w3.org/TR/wai-aria-1.1/#aria-selected
> Used in Roles: ... option 

ARIA Authoring Practices 1.1
See under the combobox "WAI-ARIA Roles, States, and Properties"
https://www.w3.org/TR/wai-aria-practices-1.1/#wai-aria-roles-states-and-properties-5
> In a combobox with a listbox, grid, or tree popup, when a suggested value is visually indicated as the currently selected value, the option, gridcell, row, or treeitem containing that value has aria-selected set to true.

I've often tested combobox widgets and helped building a few ones. Based on that, I've noticed that Windows screen readers, typically NVDA (on Firefox) and JAWS (on IE11) correctly read out the highlighted option even without `aria-selected="true"`.

The problem is with Safari + VoiceOver. Even with the current versions:
- Safari 11.1 (13605.1.33.1.2)
- VoiceOver utility 8 (562.18.4)

the highlighted option is not read out at all. Absolutely nothing: resounding silence 🙂 

In all the combobox widgets I've tested, I've noticed Safari and VoiceOver really need `aria-selected="true"`. You can see it in action also in the 5 examples in the ARIA Authoring Practices 1.1: https://www.w3.org/TR/wai-aria-practices-1.1/#examples-1

Please notice the highlighted option has `aria-selected="true"` while this attribute is omitted on the other options.

However, to actually test this there's one more problem to solve: there's also the need for an `aria-owns` attribute. I've read https://github.com/facebook/draft-js/commit/7f0cab28386ec4bde8ec6289377bff9e53cd019b and I'll open a separate issue for that. In a few words, to test this I've used Draft.js v0.10.1 to bring back support for `aria-owns`.

Testing with aria-owns in place, Safari and VoiceOver don't read out anything yet. Adding aria-selected makes them announce something.

Worth noting seems there's a Safari/VoiceOver bug that makes them read out the wrong option. This was reported also here: https://core.trac.wordpress.org/ticket/38637

**Comment by juliankrispel** [comment]
@afercia 😍 omg thanks for testing that and explaining it so well. Seems like an easy change. Fancy opening a pr yourself?

**Comment by afercia** [comment]
@juliankrispel on it :)

**Comment by juliankrispel** [comment]
@afercia if u need any guidance u can reach me on the draft js slack channel.

Btw if you fancied joining trying out joining us as a maintainer

**Comment by juliankrispel** [comment]
... we‘d love to have u! Someone who deeply cares about aria functionality and tests it for work is sorely needed amongst us!
